packages/server/src/sdk/app/views/index.ts

```diff
@@ -81,6 +81,14 @@ export function isView(view: any): view is ViewV2 {
   return view.id && docIds.isViewId(view.id) && view.version === 2
 }
 
+export function isInternal(viewId: string) {
+  if (!docIds.isViewId(viewId)) {
+    return false
```

Comment:
Should this not be an exception?

Reply:
Don't actually need this anymore - so just removed it!